Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM python:3.6.3-jessie
+FROM python:3.6.11-buster
 
 EXPOSE 80
 EXPOSE 443
 
 RUN \
 	apt-get update; \
 	apt-get install -y nginx postgresql-client mdbtools vim tidy less gettext; \
-	apt-get install -y cron --no-install-recommends
+	apt-get install -y cron --no-install-recommends; \
+	apt-get install -y binutils libproj-dev gdal-bin
```

Comment:
@geohacker Hey nothing major but could you also change `;` to `&&`
There might be cases where one of the line (7-9) throws error but last one doesn't.